src/jit/gentree.cpp

```diff
@@ -5819,20 +5811,35 @@ bool GenTree::OperMayThrow()
 
             break;
 
+        case GT_CALL:
+
+            CorInfoHelpFunc helper;
+            helper = comp->eeGetHelperNum(this->AsCall()->gtCallMethHnd);
+            if ((helper == CORINFO_HELP_UNDEF) || !comp->s_helperCallProperties.NoThrow(helper))
```

Comment:
Why not just
```
return (helper == CORINFO_HELP_UNDEF) || !comp->s_helperCallProperties.NoThrow(helper);
```
?

Reply:
No reason. I pushed a commit with an update.